### DISCUSSION TITLE
Add cursor editor support

### DISCUSSION THREAD

**Discussion by mnedoszytko** [spatie/ignition#548] [Ideas]
please add support to choose cursor https://cursor.sh/
as editor to open

suffise to add this to getDefaultOptions in IgnitionConfig.php
```
'editor_options' => [

   (...)


        'cursor' => [
                    'label' => 'Cursor',
                    'url' => 'cursor://file/%path:%line',
                    'clipboard' => false,
                ],


```

**Comment by mnedoszytko** [comment]
a small bump on this. cursor is evolving rapidly.  was just checking the flare.io also no support.

**Comment by JeppeMariagerLam** [comment]
+1

**Comment by shazeedul** [comment]
Why hasn't this been added yet?
